extensions/folder-bomb/src/move-file-out.ts

```diff
@@ -0,0 +1,7 @@
+import { ActionType, getItemAndSend } from "./send-file-utils";
+import { showToast, Toast } from "@raycast/api";
+
+export default async () => {
+  await showToast(Toast.Style.Animated, `Moving... Don't quit.`);
```

Comment:
We shouldn't really force end-users to wait until something is happening in an extension. Read my comment about background tasks.

Reply:
Hi @mattisssa Thanks for the review, I've made all the changes to the text ambiguity.

> In general, to avoid all the warnings to end-users I'd probably recommend keeping the extension in a fork from our repo or in a separate branch for a while.
As for the part you mentioned, is it that I have to close this PR and resubmit a PR to another branch?